src/internal/kopia/snapshot_manager.go

```diff
@@ -0,0 +1,232 @@
+package kopia
+
+import (
+	"context"
+	"sort"
+
+	"github.com/kopia/kopia/repo/manifest"
+	"github.com/kopia/kopia/snapshot"
+	"github.com/pkg/errors"
+
+	"github.com/alcionai/corso/src/pkg/logger"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+const (
+	// Kopia does not do comparisons properly for empty tags right now so add some
+	// placeholder value to them.
+	defaultTagValue = "0"
+	// Kopia CLI prefixes all user tags with "tag:"[1]. Maintaining this will
```

Comment:
Nit: easier reading.
```suggestion
	defaultTagValue = "0"

	// Kopia CLI prefixes all user tags with "tag:"[1]. Maintaining this will
```